### ISSUE TITLE
The training code produces much worse performance than the paper

### ISSUE THREAD

**Issue by SkyeLu** [csZcWu/GKMNet#3]
Hi, thanks for releasing your training code. I try to train the model following the training code you given, however, the best testing PSNR I got is only 24.986 dB (from epoch 3632), which is much worse than the performance reported in the paper. I wonder which step went wrong? Thanks.

**Comment by csZcWu** [comment]
Thanks for pointing out the problem. In the previous training code, the learning rate was mistakenly set as 1e-5. A new one has been updated to fixed this problem. Thanks again.

**Comment by SkyeLu** [comment]
Thanks a lot.
